cypress.config.js

```diff
@@ -7,12 +7,25 @@ module.exports = defineConfig({
     setupNodeEvents(on, config) {
       on('task', {
         generateUser() {
-          const email = faker.internet.email();
-          const randomNumber = Math.floor(Math.random(1000) * 1000);
+          const username = faker.person.name();
+          const email = faker.internet.email().toLowerCase();
+          const password = '12345Qwert!';
```

Comment:
Hardcoding the password is not a good practice. For better security, you can generate a random password using faker. For example: `const password = faker.internet.password();`. However, make sure that the generated password meets the requirements of the system you are testing.